docs/reference/pixi_configuration.md

```diff
@@ -0,0 +1,210 @@
+    | 5            | Command line arguments (`--tls-no-verify`, `--change-ps1=false`, etc.) | Configuration via command line arguments                                                       |
+
+!!! note
+    The highest priority wins. If a configuration file is found in a higher priority location, the lower priority locations are overwritten.
+
+
+!!! note
+    To find the locations where `pixi` looks for configuration files, run
+    `pixi` with `-v` or `--verbose`.
+
+## Reference
+
+??? info "Casing In Configuration"
+    In versions of pixi `0.20.1` and older the global configuration used snake_case
+    we've changed to `kebab-case` for consistency with the rest of the configuration.
+    But we still support the old `snake_case` configuration, for older configuration options.
+    These are:
+
+    - `default_channels`
+    - `change_ps1`
+    - `tls_no_verify`
+    - `authentication_override_file`
+    - `mirrors` and sub-options
+    - `repodata-config` and sub-options
+
+The following reference describes all available configuration options.
+
+### `default-channels`
+
+The default channels to select when running `pixi init` or `pixi global install`.
+This defaults to only conda-forge.
+```toml title="config.toml"
+default-channels = ["conda-forge"]
+```
+
+### `change-ps1`
+
+When set to false, the `(pixi)` prefix in the shell prompt is removed.
+This applies to the `pixi shell` subcommand.
+You can override this from the CLI with `--change-ps1`.
+
+```toml title="config.toml"
+change-ps1 = true
+```
+
+### `tls-no-verify`
+When set to true, the TLS certificates are not verified.
+
+!!! warning
+
+    This is a security risk and should only be used for testing purposes or internal networks.
+
+You can override this from the CLI with `--tls-no-verify`.
+
+```toml title="config.toml"
+tls-no-verify = false
+```
+
+### `authentication-override-file`
+Override from where the authentication information is loaded.
+Usually, we try to use the keyring to load authentication data from, and only use a JSON
+file as a fallback. This option allows you to force the use of a JSON file.
+Read more in the authentication section.
+```toml title="config.toml"
+authentication-override-file = "/path/to/your/override.json"
+```
+
+### `target-environment-directory`
+The directory where pixi stores the project environments, what would normally be placed in the `.pixi` folder in a project's root.
+It doesn't affect the environments build with `pixi global` as they listen to the `PIXI_HOME` environment variable.
+This is not recommended to change, but can be useful in some cases.
+
+- Forcing the installation on a specific filesystem/drive
+- Network hosted projects, but local environments
+- Let the system-administrator have more control over all environments on a system.
```

Comment:
```suggestion
!!! warning
    We recommend against using this because any environment created for a project is no longer placed in the same folder as the project. 
    This creates a disconnect between the project and its environments and manual cleanup of the environments is required when deleting the project.
    However, in some cases, this option can still be very useful, for instance to:

    - force the installation on a specific filesystem/drive.
    - install environments locally but keep the project on a network drive.
    - let a system-administrator have more control over all environments on a system.
```